app/init/global_event_handler.js

```diff
@@ -161,13 +166,8 @@ class GlobalEventHandler {
         const cacheDir = RNFetchBlob.fs.dirs.CacheDir;
         const mainPath = cacheDir.split('/').slice(0, -1).join('/');
 
-        try {
-            await RNFetchBlob.fs.unlink(cacheDir);
-        } catch (e) {
-            console.log('Failed to remove cache folder', e); //eslint-disable-line no-console
-        }
-
```

Comment:
No longer needed because of new use of FastImage?

Reply:
no longer needed as this happens in the `deleteFileCache` function